launcher/minecraft/mod/tasks/ModFolderLoadTask.h

```diff
@@ -48,7 +48,7 @@ class ModFolderLoadTask : public QObject, public QRunnable
     Q_OBJECT
 public:
     struct Result {
-        QMap<QString, Mod> mods;
+        QMap<QString, Mod*> mods;
```

Comment:
LeakBGone : raw pointers = cringe 
```suggestion
        QMap<QString, Mod::Ptr> mods;
```